app/views/volunteers/region_admin.html.erb

```diff
@@ -127,3 +129,7 @@
     <% end %>
   </div>
 </div>
+
+<%= javascript_include_tag "datepicker_footer" %>
+
+<%= "\n" %>
```

Comment:
Hmmm what's this? If you added this because I asked for a line break, I just meant a line break like pushing enter on the keyboard. 

So I'd take line 135 out and leave 134 in so there is an empty line at the end of the file. 

Sorry for that confusion!

Reply:
Yea, added because you asked for a line break. 

My first reaction was to just press enter, but my text editor (Atom) just erased it as soon as I hit enter. Maybe it's my text editor? I'll try in VS Code or Sublime.